doc/spire_agent.md

```diff
@@ -37,20 +37,22 @@ SPIRE configuration files may be represented in either HCL or JSON. Please see t
-| `socket_path`        | Location to bind the workload API socket                              | $PWD/spire_api       |
-| `trust_bundle_path`  | Path to the SPIRE server CA bundle                                    |                      |
-| `insecure_bootstrap` | If true, the agent bootstraps without verifying the server's identity | false                |
-| `trust_domain`       | The trust domain that this agent belongs to                           |                      |
-| `join_token`         | An optional token which has been generated by the SPIRE server        |                      |
-| `enable_sds`         | Enables [Envoy SDS support](#envoy-sds-support)                       | false                |
+| Configuration                         | Description                                                           | Default              |
+| ------------------------------------- | --------------------------------------------------------------------- | -------------------- |
+| `data_dir`                            | A directory the agent can use for its runtime data                    | $PWD                 |
+| `log_file`                            | File to write logs to                                                 |                      |
+| `log_level`                           | Sets the logging level \<DEBUG\|INFO\|WARN\|ERROR\>                   | INFO                 |
+| `log_format`                          | Format of logs, \<text\|json\>                                        | Text                 |
+| `server_address`                      | DNS name or IP address of the SPIRE server                            |                      |
+| `server_port`                         | Port number of the SPIRE server                                       |                      |
+| `socket_path`                         | Location to bind the workload API socket                              | $PWD/spire_api       |
+| `trust_bundle_path`                   | Path to the SPIRE server CA bundle                                    |                      |
+| `insecure_bootstrap`                  | If true, the agent bootstraps without verifying the server's identity | false                |
+| `trust_domain`                        | The trust domain that this agent belongs to                           |                      |
+| `join_token`                          | An optional token which has been generated by the SPIRE server        |                      |
+| `enable_sds`                          | Enables [Envoy SDS support](#envoy-sds-support)                       | false                |
+| `sds_default_tls_certificate_name`    | SDS default TLS certificate resource name                             |                      |
+| `sds_default_validation_context_name` | SDS default validation context resource name                          |                      |
```

Comment:
How about using SPIRE terminology here? It will be shorter, and slightly more accurate (e.g. we are delivering an X509-SVID, not just any tls certificate). The description could then translate to SDS-speak. Perhaps something like this?

```suggestion
| `sds_default_svid_name`    | The TLS Certificate resource name to use for the default X509-SVID with Envoy SDS                             |                      |
| `sds_default_bundle_name` | The Validation Context resource name to use for the default X.509 bundle with Envoy SDS                          |                      |
```